### PR TITLE
create python-publish.yml

### LINKED ISSUE #41: rename coronavirus -> oscovida, put on PyPI
- [x] Rename the current package name 'coronavirus' to 'oscovida'
- [x] update code and tests
- [x] update notebooks in dev, tools and notebooks 
- [ ] put on PyPI

### PR DESCRIPTION
**Pull request by kirienko**
This Github action will be triggered on a release.

@fangohr I'm sending you sensitive information by email.

Closes #41

### CODE REVIEW

**Comment by kirienko** [comment]
Tests are failing, but it's not me :man_shrugging:

**Comment by kirienko** [comment]
Now tests are passing, thanks to that fix in Travis #60.

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```
Comment (RobertRosca) [inline .github/workflows/python-publish.yml:22]: Huh neat, didn't know you could do `3.x` to pick the highest version.

Comment (kirienko) [reply .github/workflows/python-publish.yml:22]: I can put `3.8` here, I guess, though I doubt we use anything specific from `3.7` or `3.8`, don't we? All in all, it should work on `3.6` regardless of that.

**Pull request merged**